### ISSUE TITLE
mess with pages with # in url

### ISSUE THREAD

**Issue by seyyed-sina** [Easyfood/pageAccelerator#15]
I use custombox.js modal in my page and it uses ID and href=#id to show modal on page, seems it has conflict between these addressing in URL, cause it stop page to show content correctly

**Comment by EvandroLG** [comment]
It's really a problem and I will solve it ;)
For now, you can add on that links a `data-attribute="false"`, so pageAccelerator won't follow that links.

Let me know if it fix your problem @spostad7

**Comment by seyyed-sina** [comment]
It still has problem!
I figured out that it has problem with all anchor tag I used for some purpose, for example trigger a slideDown menu or go to top arrow, although I used data-attribute="false"

**Comment by EvandroLG** [comment]
ops, so sorry. the correct would be data-pageAccelerator="false" :)

**Comment by seyyed-sina** [comment]
no luck!
it had better performance on triggering modal, but when i close the modal, the page content stop showing again,
and also the problem with anchor tag still the same cause I didn't addressed in href.. (i used # in href).
I have to say that I used transition and opacity to showing page content on load, and for the first time it successfully show the content, but when I click `<a href="#something"></a>` it stop the content to show. beside I checked the inspect element and DOM didn't changed for displaying issue
